kodi_addon_checker/check_addon.py

```diff
@@ -61,7 +61,7 @@ def start(addon_path, branch_name, all_repo_addons, pr, config=None):
 
             check_files.check_for_invalid_json_files(addon_report, file_index)
 
-            check_artwork.check_artwork(addon_report, addon_path, parsed_xml, file_index)
+            check_artwork.check_artwork(addon_report, addon_path, parsed_xml, file_index, branch_name)
```

Comment:
This needs a rebase on top of master and the following change.
```suggestion
            check_artwork.check_artwork(addon_report, addon_path, parsed_xml, file_index, args.branch)
```